docs/_documentations/container-registry-credentials.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: docs
+title: Container registry credentials for Codewind
+description: Specifying container registries for Codewind
+keywords: docker, containers, registry, Codewind, Appsody
+duration: 5 minutes
+permalink: containerregistry
+type: document
+parent: installoncloud
+order: 1
+---
+# Specifying Container Registries for Codewind 
+
+When you run Codewind on Kubernetes, Codewind uses [`buildah`](https://github.com/containers/buildah) to build container images from Dockerfiles and then push the images to a container registry that you specify.
```

Comment:
Add a sentence, something like `You can use the Image Registry Manager in your Codewind IDE to configure the registry to use`